src/domain/event/eventContent/EventContent.tsx

```diff
@@ -22,6 +22,8 @@ const EventContent: React.FC<Props> = ({ event, eventType }) => {
   const locale = useLocale();
   const { description, photographerName } = getEventFields(event, locale);
 
+  const isInternetEvent = event?.location?.id === 'helsinki:internet';
```

Comment:
`'helsinki:internet'` could be stored as a constant, might be used somewhere else in the future.

Reply:
fixed